### ISSUE TITLE
Space: Vending machine

### ISSUE THREAD

**Issue by sutantyo** [sutantyo/LearningSpaceSurvey#10]
If it is possible, it would be nice to be able to get food and drink after hours

**Comment by amita-trikam** [comment]
Walking to and from BD building is very far for food, vending machines for quick bites/drinks would be nice. Or at the very least, a drinking fountain/bubbler to refill water bottles.

**Comment by sutantyo** [comment]
we plan to have a kicthenette of some sort with water filling station

**Comment by ghost** [comment]
> we plan to have a kicthenette of some sort with water filling station

Yep thats a need

**Comment by mqlnx** [comment]
Speaking of water refill, can we also have a hot water tap? I usually bring my own tea or coffee on campus to save money

**Comment by sutantyo** [comment]
Yeah, definitely on hot water tap. We probably gonna have those hot/cold tap you see in some places

**Comment by snc-89** [comment]
a little kitchenette would be nice
